webextensions/api/declarativeNetRequest.json

```diff
@@ -139,6 +139,25 @@
             }
           }
         },
+        "MAX_NUMBER_OF_DISABLED_STATIC_RULES": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
Add note that although the constant is not exposed, that Chrome enforces a limit, AND that the limit is counted across all static rulesets instead of per ruleset.

The "not exposed' part is intentional for now, per https://github.com/w3c/webextensions/issues/162#issuecomment-2211076093